app/_kong_plugins/header-cert-auth/index.md

```diff
@@ -36,6 +36,144 @@ search_aliases:
+
+Additionally, the plugin has a [static priority](/gateway/entities/plugin/#plugin-priority) so that it runs after all authentication plugins, allowing other auth plugins (for example, [Basic Auth](/plugins/basic-auth/)) to secure the source first. This ensures that the source is secured by multiple layers of authentication by providing L7 level of security.
+
+## Header size
+
+Sending certificates in headers may exceed header size limits in some environments. 
+You can configure {{site.base_gateway}} to accept larger headers by configuring the [Nginx header buffer parameter in `kong.conf`](/gateway/configuration/#nginx-http-large-client-header-buffers). 
+For example:
+
+```
+nginx_proxy_large_client_header_buffers=8 24k
+```
+
+Or via an environment variable:
+```
+KONG_NGINX_PROXY_LARGE_CLIENT_HEADER_BUFFERS=8 24k
+```
+
+## Client certificate request
+
+The `send_ca_dn` option is not supported in this plugin. This is used in mutual TLS authentication, where the server sends the list of trusted CAs to the client, and the client then uses this list to select the appropriate certificate to present. In this case, since the plugin doesn't do TLS handshakes and only parses the client certificate from the header, it isn't applicable.
+
+The same applies to SNI functionality. The plugin can verify the certificate without needing to know the specific hostname or domain being accessed. The plugin's authentication logic is decoupled from the TLS handshake and SNI, so it doesn't need to rely on SNI to function correctly (pretty much anything that deals with the actual TLS handshake is not needed for the plugin to work).
+
+## Manual mappings between Certificate and Consumer objects
+
+Sometimes, you might not want to use automatic Consumer lookup, or you have certificates
+that contain a field value not directly associated with Consumer objects. In those
+situations, you may manually assign one or more subject names to the [Consumer entity](/gateway/entities/consumer/) for
+identifying the correct Consumer.
+
+{:.info}
+> **Note**: Subject names refer to the certificate's Subject Alternative Names (SAN) or
+Common Name (CN). CN is only used if the SAN extension does not exist.
+
+You can create a Consumer mapping with either of the following:
+  * The [`/consumers/{consumer}/header-cert-auth` Admin API endpoint](/api/gateway/admin-ee/#/operations/create-plugin-for-consumer)
+  * [decK](/gateway/entities/consumer/#set-up-a-consumer) by specifying `header_cert_auth_credentials` in the configuration like the following:
+    
+    ```yaml
+    consumers:
+    - custom_id: my-consumer
+      username: {consumer}
+      header_cert_auth_credentials:
+      - id: bda09448-3b10-4da7-a83b-2a8ba6021f0c
+        subject_name: test@example.com
+    ```
+
+The following table describes how the mapping parameters work:
+
+| Form Parameter                            | Default | Description |
```

Comment:
You can/should use the `{% table %}` tag here instead. Eg see https://github.com/Kong/developer.konghq.com/blob/main/app/_kong_plugins/ai-rate-limiting-advanced/index.md?plain=1#L121.